frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -198,6 +199,30 @@ export default class FirebaseInteractor {
     });
   }
 
+  async createInterventionFromAssessment(
+    responses: AssessmentResult[]
+  ): Promise<void> {
+    let incorrectWords = responses.filter((response) => !response.correct);
+    shuffle(incorrectWords);
+    for (let i = 0; i < 8; i++) {
+      // Add 3 words per intervention (wrapping around for now)
+      let wordList = [
+        incorrectWords[(i * 3) % incorrectWords.length].word,
+        incorrectWords[(i * 3 + 1) % incorrectWords.length].word,
+        incorrectWords[(i * 3 + 2) % incorrectWords.length].word,
+      ];
+      await this.db.collection("interventions").add({
+        activityIdx: 0,
+        wordIdx: 0,
+        wordList,
+        // Assign it to the current user
+        userId: this.auth.currentUser?.uid,
+        // Decide which session the intervention is in
+        interventionSession: i,
```

Comment:
nit: the naming `InterventionSession` is a bit confusing here since it makes me think of auth sessions. Would something like `interventionGrouping` make more sense?

Reply:
I think just `session` would also work. Do you agree?